guides/hhvm/02-basic-usage/03-server.md

```diff
@@ -38,9 +47,8 @@ http://your.site:8080/index.php
 
 You can also use `curl` and other programs to access the HHVM server as well.
 
-### Possible Fatal Error
-
-If the code you are running is written in [Hack (<?hh)](/hack/) and you run into a [fatal error regarding not running the typechecker](/hhvm/FAQ/faq#running-code__how-do-i-fix-the-not-running-the-hack-typechecker-fatal-error), then you must do one of the following:
+## Debugging a Fatal Error
```

Comment:
The default value for `hhvm.hack.lang.look_for_typechecker` has been false for a _really_ long time. I don't know if this is still worth mentioning. Earlier hhvm versions (around 3.23 iirc) would refuse to run any Hack code if the typechecker was not running.

All code for hhvm has been Hack since 4.0, PHP is no longer supported. This sentence is an historical note only.

Reply:
@lexidor Thanks! That part did feel weird when I read it. Glad you pointed it out.

Reply:
@lexidor In that case, the preceding section should probably say something like:

`http://your.site:8080/index.hack` instead of `http://your.site:8080/index.php` (for an example of a client request to the server), right?

Reply:
Hack code can still be written in dot-php files. But we should indeed encourage dot-hack everywhere.